TimePicker/TimePickerBase.js

```diff
@@ -211,7 +211,7 @@ const TimePickerBase = kind({
 
 	computed: {
 		hasMeridiem: ({order}) => order.indexOf('a') >= 0,
-		meridiemPickerWidth: ({meridiem, meridiems}) => meridiems[meridiem].length * 2
+		meridiemPickerWidth: ({meridiem, meridiems}) => meridiems[meridiem].length
```

Comment:
Let's reduce this to 1.5.
Without the multiplier, the width varies too much when the meridem changes in am-ET locale.